cmd/git-init/main.go

```diff
@@ -66,10 +68,30 @@ func main() {
 		logger.Fatalf("Unexpected error creating symlink: %v", err)
 	}
 
+	dir, err := os.Getwd()
+	if err != nil {
+		logger.Errorf("Failed to get current dir", err)
+	}
+
+	if *name != "" {
```

Comment:
This all seems largely equivalent to:

```golang
if len(*name) > 0 {
  run(logger, "git", "init", *name)
  os.Chdir(*name)
} else {
  run(logger, "git", "init")
}
```

Reply:
yes agreed 100%. I like this better and its so simple :D